index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>velog clone</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=BioRhyme&family=Hepta+Slab:wght@300;500&family=Inconsolata&family=Noto+Sans+KR:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <span id="logo">velog</span>
+    </header>
+    
+    <div class="sort">
+        <button id="sortBytrend">트렌딩</button>
+        <button id="sortBylatest">최신</button>
+    </div>
```

Comment:
<button> 태그 대신에 <nav> 태그를 사용하는 것도 좋을 것 같아요!!

```suggestion
    <div class="sort">
        <nav>
             <ul>
                  <li><a href="#">트렌딩</a></li> 
                  <li><a href="#">최신</a></li>
             </ul>
        </nav>        
    </div>
```